pkg/spec/resources_fix.go

```diff
@@ -110,7 +71,7 @@ func fixConfigMaps(configMaps []ConfigMapMod, appName string) ([]ConfigMapMod, e
 		// if multiple configMaps is defined then each should have a name
 		for cdn, cd := range configMaps {
 			if cd.Name == "" {
-				return nil, fmt.Errorf("name not specified for app.configMaps[%d]", cdn)
+				return nil, fmt.Errorf("name not specified for deployment.configMaps[%d]", cdn)
```

Comment:
This will still be `app`, because of the following reasons:

- this is not tied to deployment in anyway
- user will still be considering it as app
- the error is user facing

Reply:
changed